examples/manifold/plot_lle_digits.py

```diff
@@ -51,17 +51,17 @@
 def plot_embedding(X, title, ax):
     X = MinMaxScaler().fit_transform(X)
 
+    for digit in digits.target_names:
+        ax.scatter(
+            *X[y == digit].T,
+            marker=f"${digit}$",
+            s=60,
+            color=plt.cm.Dark2(digit),
+            alpha=0.7,
+        )
```

Comment:
Does this addition (and the deletion bellow) change the rendering of this example?

Reply:
Default plot
![previous](https://user-images.githubusercontent.com/25286805/143011874-66252390-e873-483e-923c-0edaac76a0e8.png)

after this changes
![newlook](https://user-images.githubusercontent.com/25286805/143012002-8bf55d9c-19c8-49ca-8fb5-005429bd3db8.png)

changing zorder:
![changedZorder](https://user-images.githubusercontent.com/25286805/143012067-4eea3ecf-4565-4c63-8fe9-4c13d3fdfbb4.png)